assets/src/stories-editor/helpers/index.js

```diff
@@ -1582,3 +1607,18 @@ export const uploadVideoFrame = async ( { id: videoId, src } ) => {
 		} );
 	} );
 };
+
+/**
+ * Add anchor for a block if it's missing.
+ *
+ * @param {string} clientId Block ID.
+ */
+export const maybeAddMissingAnchor = ( clientId ) => {
```

Comment:
Isn't that a duplicate of `conditionallyUpdateAnchor`? I think we should remove all usage of `conditionallyUpdateAnchor` with this new function as it has a better name :-)

Reply:
Actually that does something different. 🤦‍♂ 

Nevermind.